source/renderer/app/components/profile/bug-report/BugReportDialog.js

```diff
@@ -302,8 +307,8 @@ export default class BugReportDialog extends Component<Props, State> {
         title={intl.formatMessage(messages.title)}
         actions={!error ? actions : alternativeActions}
         closeOnOverlayClick
-        onClose={onCancel}
-        closeButton={<DialogCloseButton onClose={onCancel} />}
+        onClose={this.onClose}
+        closeButton={<DialogCloseButton disabled={isSubmitting && !error} onClose={this.onClose} />}
```

Comment:
@daniloprates closing should be possible in case of an error - we should prevent it only in case of submitting state!